test/lender/Lender.sol

```diff
@@ -26,6 +26,8 @@ contract Lender {
   /// @param s: the swivel contract
   /// @param p: the pendle contract
   /// @param t: the tempus contract
+  /// @dev the constrcutor sets a default value for the feenominator, which 
```

Comment:
the pattern in swivel was that @dev went above @param  (and @notice above that)

    @notice ...
    @dev ...
    @param ...
    @param...